src/rrd_graph.c

```diff
@@ -6162,7 +6162,7 @@ image_title_t graph_title_split(
     image_title_t retval;
     char *str;
     int count = 0;
-    const char delim[2] = "?";
+    const char delim[2] = "\n";
 
```

Comment:
you could make your code handle both ... a chr(10) as well as a literal '\n' in that way it would work regardless if there is an interpreter in the path that already evaluates \n or not.

Reply:
Oh I see what you mean.  In the same way you'd format a printf string.